lms/djangoapps/enterprise/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from entitlements.models import EntitlementGroup
+
+
+class EnterpriseCustomer(models.Model):
+    name = models.CharField(max_length=255)
+    entitlement_group = models.OneToOneField(EntitlementGroup, related_name="enterprise_customer")
+    linked_third_party_providers = models.CharField(
```

Comment:
Usage of `CharField` is clearly suboptimal here - it becomes a responsibility of manager to make sure (1) the slug is correct and (2) the slug is unique and not used by other providers.

Ideally it should be a ForeignKey with unique=True, but the problem is that TPA app is optional (can be disabled by feature flag). In that case there won't be a model to reference (and a DB table), so likely it will just crash on startup. So I've decided to make it just CharField for the prototyping purposes, but later we might want something more robust.